### DISCUSSION TITLE
Amazon S3 authentication

### DISCUSSION THREAD

**Discussion by rp-ryanwilson** [nextgenhealthcare/connect#6359] [Q&A]
For connectivity to certain s3 buckets we need to specify an access key ID, secret access key and an AWS session token.  How do we specify the token value when connecting through the File Reader/Writer Amazon S3 configuration?

**Comment by JoeFox82** [comment]
Look at the post here, perhaps this can help you: https://github.com/nextgenhealthcare/connect/discussions/5462

**Reply by rp-ryanwilson** [reply]
I had reviewed that post, but I don't see anything in there that specifically helps.  Our security team wants us to authenticate to S3 storage using the tokens in addition to the other two keys as the tokens regenerate every hour.  But I need to figure out how to be able to use the token in addition to the other two to get connectivity to work from Mirth.
